File: javaTutionCode/8th_june/pr1.c
#include<stdio.h>
void main(){

  printf("%ld\n",sizeof(int));  //4
  printf("%ld\n",sizeof(long));//8
  printf("%ld\n",sizeof(char));//1
  printf("%ld\n",sizeof(float));//4
  printf("%ld\n",sizeof(double));//8
  printf("%ld\n",sizeof(_Bool));//1

}
/*
Output
abhijeetzgd@abhijeetzgd-ThinkPad-E490:~/Java/javaTutionCode/6th_june$ cc pr1.c
abhijeetzgd@abhijeetzgd-ThinkPad-E490:~/Java/javaTutionCode/6th_june$ ./a.out
4
8
1
4
8
1
*/

File: MyPractice/Methods/pr1.java
/*calling a instance method without parameter*/
class Addition{

  int num1=20,num2=10;//instance Variables
// claculate addition of two number instance method
  void sum(){
    int sum=num1+num2;
    System.out.println(" sum = "+sum);
  }
}

class Method{

  public static void main(String [] args){

    Addition obj = new Addition();//creating object of class addition
    obj.sum();//calling sum method
  }
}
/*
output
abhijeetzgd@abhijeetzgd-ThinkPad-E490:~/Java/MyPractice/staticMethod$ javac pr1.java
abhijeetzgd@abhijeetzgd-ThinkPad-E490:~/Java/MyPractice/staticMethod$ java Method
 sum = 30
*/
